keras_nlp/models/bert/bert_preprocessing.py

```diff
@@ -185,9 +171,88 @@ def call(self, inputs):
             "padding_mask": token_ids != self.pad_token_id,
         }
 
+    @classmethod
+    def from_preset(
+        cls,
+        preset,
+        sequence_length=None,
+        truncate="round_robin",
+        **kwargs,
+    ):
+
+        if preset not in Bert.presets:
+            raise ValueError(
+                "`preset` must be one of "
+                f"""{", ".join(Bert.presets)}. Received: {preset}."""
+            )
+        metadata = Bert.presets[preset]
+
+        vocabulary = _download_vocabulary(
+            preset,
+            metadata["vocabulary_url"],
+            metadata["vocabulary_hash"],
+        )
+        config = metadata["preprocessor_config"]
+        # Use model's `max_sequence_length` if `sequence_length` unspecified;
+        # otherwise check that `sequence_length` not too long.
+        max_sequence_length = metadata["config"]["max_sequence_length"]
```

Comment:
Adding a `presets` property to every class of compatible ids sounds like a reasonable idea. Done for this class!

Re: config, there is a lot of work behind the scenes to get the inputs for the generic constructor. I don't think it can be very clean with `vocabulary` as an argument.